M2/Macaulay2/d/Makefile.in

```diff
@@ -210,27 +206,30 @@ ifeq (@HAVE_WFRAME_ADDRESS@,yes)
 M2lib.o : M2_CFLAGS += -Wno-frame-address
 endif
 
+# flags needed to sidestep system regex
+scclib.o : CPPFLAGS += -Dre_compile_fastmap=M2_re_compile_fastmap -Dre_compile_pattern=M2_re_compile_pattern -Dre_match=M2_re_match -Dre_match_2=M2_re_match_2 -Dre_search=M2_re_search -Dre_search_2=M2_re_search_2 -Dre_set_registers=M2_re_set_registers -Dre_set_syntax=M2_re_set_syntax -Dregcomp=M2_regcomp -Dregerror=M2_regerror -Dregexec=M2_regexec -Dregfree=M2_regfree
```

Comment:
Yes; I have a branch where regex.h from system is used instead of Macaulay2/regex. Putting the macros in Makefile allows having an option to do that.

Reply:
It would be better to have one command line macro definition, such as "M2_REGEX", to trigger the others, to keep the command line less crowded, and to have that long list of names in just one place, rather than in two places (a make file and a cmake file).

Reply:
Can we postpone this to #1092? The faster this is merged, the faster I can get to switching to a better regex engine.

Reply:
Okay, but don't forget.